ui/src/tempVars/constants/index.ts

```diff
@@ -0,0 +1,168 @@
+import uuid from 'uuid'
+
+import {TimeRange} from 'src/types/query'
+import {TEMP_VAR_DASHBOARD_TIME} from 'src/shared/constants'
+import {Template, TemplateType, TemplateValueType} from 'src/types'
+
+export const TEMPLATE_TYPES_LIST = [
+  {
+    text: 'Databases',
+    type: TemplateType.Databases,
+  },
+  {
+    text: 'Measurements',
+    type: TemplateType.Measurements,
+  },
+  {
+    text: 'Field Keys',
+    type: TemplateType.FieldKeys,
+  },
+  {
+    text: 'Tag Keys',
+    type: TemplateType.TagKeys,
+  },
+  {
+    text: 'Tag Values',
+    type: TemplateType.TagValues,
+  },
+  {
+    text: 'CSV',
+    type: TemplateType.CSV,
+  },
+]
+
+export const TEMPLATE_VARIABLE_TYPES = {
+  [TemplateType.CSV]: TemplateValueType.CSV,
+  [TemplateType.Databases]: TemplateValueType.Database,
+  [TemplateType.Measurements]: TemplateValueType.Measurement,
+  [TemplateType.FieldKeys]: TemplateValueType.FieldKey,
+  [TemplateType.TagKeys]: TemplateValueType.TagKey,
+  [TemplateType.TagValues]: TemplateValueType.TagValue,
+}
+
+export const TEMPLATE_VARIABLE_QUERIES = {
```

Comment:
are these still being used by all of the components? seems like the instances where they're variables, maybe they could be functions here?